tas-client/test/TasApiFeatureProvider.test.ts

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AxiosError, AxiosPromise, AxiosResponse } from 'axios';
+import { IExperimentationTelemetry } from 'src';
```

Comment:
Is this the right import? `from 'src'`?

Reply:
this was auto-imported by vscode and didn't realize the import was wrong, I've fixed it. good catch!